src/localize/languages/pt-pt.json

```diff
@@ -0,0 +1,467 @@
+      "preload": "Pré-carregar a visualização ao vivo em segundo plano",
+      "show_image_during_load": "Mostar imagem durante o carregamento",
+      "transition_effect": "Efeito de transição de câmera ao vivo"
+    },
+    "media_viewer": {
+      "auto_mute": "Silenciar mídia automaticamente",
+      "auto_pause": "Pausar mídia automaticamente",
+      "auto_play": "Reproduzir mídia automaticamente",
+      "auto_unmute": "Ativar mídia automaticamente",
+      "controls": {
+        "editor_label": "Controles do visualizador de mídia"
+      },
+      "draggable": "Visualizador de eventos pode ser arrastado/deslizado",
+      "lazy_load": "A mídia do Visualizador de eventos é carregada lentamente no carrossel",
+      "layout": "layout",
+      "transition_effect": "Efeito de transição do Visualizador de eventos",
+      "transition_effects": {
+        "none": "Sem transição",
+        "slide": "Transição de slides"
+      }
+    },
+    "menu": {
+      "alignment": "Alinhamento do menu",
+      "alignments": {
+        "bottom": "Alinhado à parte inferior",
+        "left": "Alinhado à esquerda",
+        "right": "Alinhado à direita",
+        "top": "Alinhado ao topo"
+      },
+      "button_size": "Tamanho do botão de menu (e.g. '40px')",
+      "buttons": {
+        "alignment": "Alinhamento do botão",
+        "alignments": {
+          "matching": "Mesmo alinhamento do menu",
+          "opposing": "Opor-se ao alinhamento do menu"
+        },
+        "camera_ui": "",
+        "cameras": "Selecionar câmera",
+        "clips": "Clipes",
+        "download": "Baixe a mídia do evento",
+        "enabled": "Botão ativado",
+        "expand": "expandir",
+        "frigate": "Frigate menu / Visualização padrão",
+        "fullscreen": "Tela cheia",
+        "icon": "Ícone",
+        "image": "Imagem",
+        "live": "Ao vivo",
+        "media_player": "Enviar para o reprodutor de mídia",
+        "priority": "Prioridade",
+        "substreams": "substreams",
+        "snapshots": "Instantâneos",
+        "timeline": "Linha do tempo"
+      },
+      "position": "Posição do menu",
+      "positions": {
+        "bottom": "Posicionado na parte inferior",
+        "left": "Posicionado à esquerda",
+        "right": "Posicionado à direita",
+        "top": "Posicionado no topo"
+      },
+      "style": "Estilo do menu",
+      "styles": {
+        "hidden": "Menu oculto",
+        "hover": "Menu suspenso",
+        "none": "Sem menu",
+        "outside": "Menu externo",
+        "overlay": "Menu sobreposto"
+      }
+    },
+    "overrides": {
+      "info": "Esta configuração do cartão especificou manualmente as substituições configuradas que podem substituir os valores mostrados no editor visual, consulte o editor de código para visualizar/modificar essas substituições"
+    },
+    "performance": {
+      "warning": "Avisos",
+      "features": {
+        "editor_label": "editor_label",
```

Comment:
(Another example of where the value very likely should not equal the key -- as this will literally show 'editor_label' in the UI, which is almost certainly incorrect).